### DISCUSSION TITLE
[ERROR GPL-0304] RePlAce diverged at initial iteration. Please tune the parameters again

### DISCUSSION THREAD

**Discussion by dineshannayya** [The-OpenROAD-Project/OpenROAD-flow-scripts#55] [Q&A] [answered]
I am trying to do Global Placement setup through Openlane flow.  This is small design with 6500 cells. Tool exit with below

**[ERROR GPL-0304] RePlAce diverged at initial iteration. Please tune the parameters again**

Error message does not indicate which parameter need to changed,  I trying playing with DIE Area and Density .. It does not solves the issue..  Increasing the DIE Size tool crashes with Segmentation fault.  Can any one suggest step to solve the issue.

FYI: Flow worked well with partition with 20K to 60K design without any issue.

**Comment by maliberty** [comment]
in the current OR the message is

    log_->error(GPL, 304, "RePlAce diverged at initial iteration. "
       "Re-run with a smaller init_density_penalty value.");

**Reply by vijayank88** [reply, marked as answer]
With latest OpenROAD head, mostly these error won't repeats. Any corner case, feel free to open an issue.
